docs/user-docs/latest/plugins/installing-plugins-via-marketplace.md

```diff
@@ -4,37 +4,39 @@ description: Instructions to install new plugins in a Strapi application via the
 canonicalUrl: https://docs.strapi.io/user-docs/latest/plugins/installing-plugins-via-marketplace.html
 ---
 
-# Installing plugins via the Marketplace
+# Managing Marketplace plugins
 
-::: callout 🚧 The Marketplace is currently not available for v4. It will be back soon, stay tuned!
-In the meantime, plugins can be installed via the Command Line Interface (see [Developer Documentation](/developer-docs/latest/developer-resources/cli/CLI.md#strapi-install)).
-:::
-
-The Marketplace is a section of the admin panel that lists all plugins that can be installed in a Strapi application. The Marketplace is accessible from ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_ in the main navigation of the admin panel.
-
-<!--
+The Marketplace is where users can find additional plugins to customize Strapi applications. The Marketplace is located in the admin panel, indicated by ![Marketplace icon](../assets/icons/marketplace.svg) _Marketplace_. In the Marketplace, users can browse or search for plugins, link to detailed plugin descriptions and submit new plugins. This documentation describes how to navigate the Marketplace and how to install Marketplace plugins.
 
+::: caution
+ v3 and v4 plugins are not cross-compatible.
+:::
+<!-- Is it better to keep these two callouts separate or combine them into the caution callout? -->
```

Comment:
Actually I think you're right, it looks weird to have those 2 separated 🤔  Also, now that I re-read everything, it doesn't make sense to have a big caution callout since the in-app Marketplace only displays v4 plugins anyways.
So perhaps what we could do is create a Strapi callout (the purple one) to "promote" the Strapi Market. For example:
```
::: strapi
The Marketplace in the admin panel only displays v4 plugins, but all plugins for all Strapi versions are discoverable in the Strapi Market. Keep in mind however that v3 and v4 plugins are not cross-compatible.
:::
```
Or something like that. What do you think?